��serversideamitv1.py

```diff
@@ -0,0 +1,25 @@
+import time
+
+def countLogsByKeyword():
+    try:
+        keywords = ["INFO", "WARN", "ERROR"]
+        counts = {key: 0 for key in keywords}
+        with open("/var/log/syslog", "r") as log_file:
+            for line in log_file:
+                words = line.split()
+                for keyword in keywords:
+                    if keyword in words:
+                        counts[keyword] += 1
+        return counts
+    except Exception as e:
+        print("An error occurred while reading the log file: " + str(e))
+        return {key: 0 for key in ["INFO", "WARN", "ERROR"]}

```

Comment:
אם תגדיר את שורה 5 ו-6 מחוץ לטריי, תוכל להחזיר כאן 
`{key: 0 for key in keywords }`

פחות שכפול